OmniBLE/PumpManagerUI/ViewControllers/OmnipodSettingsViewController.swift

```diff
@@ -317,188 +316,188 @@ class OmnipodSettingsViewController: UITableViewController {
-//
-//        case .podDetails:
-//            let podState = self.podState!
-//            switch PodDetailsRow(rawValue: indexPath.row)! {
-//            case .podAddress:
-//                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-//                cell.textLabel?.text = LocalizedString("Assigned Address", comment: "The title text for the address assigned to the pod")
-//                cell.detailTextLabel?.text = String(format:"%04X", podState.address)
-//                return cell
-//            case .podLot:
-//                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-//                cell.textLabel?.text = LocalizedString("Lot", comment: "The title of the cell showing the pod lot id")
-//                cell.detailTextLabel?.text = String(format:"L%d", podState.lot)
-//                return cell
-//            case .podTid:
-//                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-//                cell.textLabel?.text = LocalizedString("TID", comment: "The title of the cell showing the pod TID")
-//                cell.detailTextLabel?.text = String(format:"%07d", podState.tid)
-//                return cell
-//            case .piVersion:
-//                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-//                cell.textLabel?.text = LocalizedString("PI Version", comment: "The title of the cell showing the pod pi version")
-//                cell.detailTextLabel?.text = podState.piVersion
-//                return cell
-//            case .pmVersion:
-//                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-//                cell.textLabel?.text = LocalizedString("PM Version", comment: "The title of the cell showing the pod pm version")
-//                cell.detailTextLabel?.text = podState.pmVersion
-//                return cell
-//            }
-//
-//        case .deletePumpManager:
-//            let cell = tableView.dequeueReusableCell(withIdentifier: TextButtonTableViewCell.className, for: indexPath) as! TextButtonTableViewCell
-//
-//            cell.textLabel?.text = LocalizedString("Switch from Omnipod Pumps", comment: "Title text for the button to delete Omnipod PumpManager")
-//            cell.textLabel?.textAlignment = .center
-//            cell.tintColor = .deleteColor
-//            cell.isEnabled = true
-//            return cell
-//        }
-//    }
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch sections[indexPath.section] {
+        case .status:
+            let podState = self.podState!
+            let statusRow = StatusRow(rawValue: indexPath.row)!
+            if statusRow == .alarms {
```

Comment:
To make this compile, I needed to use a different method of deriving the class name. NSStringFromClass seems to work well.